### DISCUSSION TITLE
Image normalization

### DISCUSSION THREAD

**Discussion by greenhawktobias** [azavea/raster-vision#2019] [General]
I am working on a project trying to do machine learning on scanned images. A lot of black and white, single channel images as image input data. Some images have values ​​from 0-1 and other values ​​from 0-255. But it doesn't seem to be a problem due to standard image normalization in raster vision?

**Comment by AdeelH** [comment]
Yes, RV should be able to [handle bands from different files](https://docs.rastervision.io/en/0.21/usage/tutorials/reading_raster_data.html#Combining-bands-from-different-files-using-MultiRasterSource) and with different stats. You should still make sure that all the bands look right before proceeding to training.
